README.md

```diff
@@ -1,22 +1,104 @@
-# BootCamp
-Computing Boot Camp for (new and old) KIPAC members and physics graduate students at large.
+# Welcome to KIPAC
 
-Schedule can be found on the [website](https://kipac.github.io/BootCamp).
+This page is intended to help you get started using our computing
+resources to do research while you are at KIPAC.
 
-## Dependencies
-See below for a list of dependencies. Make sure you install and test these before coming to the bootcamp!
-If you have trouble with any of these, come to Day 0 where we will help set everything up.
-### Everyone
-[Anaconda](https://conda.io/miniconda.html)
+There are several different projects ongoing at KIPAC and each of them
+has its own computing resources and environment.  However, we all do
+share certain resources.
 
-[Git](https://git-scm.com/)
+This repository started out as a place to collect materials for a
+3-day "BootCamp" for computing.   The documenation intro for that is [here](docs/BootCamp.md)
 
-### Windows Users
-[Cygwin](https://www.cygwin.com/) or another terminal emulator of your choice.
+## Getting and setting up accounts
 
-[Putty](http://www.putty.org/)
+Depending on if you are working at SLAC or on campus (or both) you
+will have computing accounts for either or both places.
+
+[SLAC computing accounts](docs/slac_computing.md)
+
+[Stanford computing accounts](docs/stanford_computing.md)
+
+
+## Setting up your computer 
+
+Depending on the project you are working on, and the details of your
+work, you may find that you do almost all or almost none of your work
+on your own computer as opposed to on SLAC or Stanford computers.
+
+However, at a minimum, you will almost certainly need to be able
+to connect terminal and graphics to remote (i.e., SLAC / Stanford)
+computers.   It is also very likely that you will need to be able to
+run Jupyter analysis notebooks both on your computer, and by
+connecting your computer to a notebook server running at SLAC.
+
+[MacOS](docs/macos_setup.md)
+
+[Windows](docs/windows_setup.md)
+
+As usual, if you are using a linux computer you are on you own.
```

Comment:
I wloud write it a bit more optimistic :), maybe state that you (the reader) can add resources if (s)he encounters problems.